manifest/x86_64/a/aide.filelist

```diff
@@ -1,6 +1,3 @@
 /usr/local/bin/aide
-/usr/local/etc/aide.conf
-/usr/local/etc/doc/aide.db
```

Comment:
What happened to the database and config?

Reply:
The config was actually a manpage, so shouldn't be installed like that, and the database was an empty file.

Reply:
What?  This is what makes this thing work.

Reply:
`aide.conf.5` is a manual page, not a configuration file. The `doc/aide.conf` file that was present in `0.15.1` is no longer there.